packages/falso/src/lib/resolution.json

```diff
@@ -0,0 +1,436 @@
+    },
+    {
+      "width": "1080",
+      "height": "2160"
+    },
+    {
+      "width": "1080",
+      "height": "2340"
+    },
+    {
+      "width": "1440",
+      "height": "3120"
+    },
+    {
+      "width": "1080",
+      "height": "2340"
+    },
+    {
+      "width": "1176",
+      "height": "2400"
+    },
+    {
+      "width": "1080",
+      "height": "2310"
+    },
+    {
+      "width": "1080",
+      "height": "2340"
+    },
+    {
+      "width": "1080",
+      "height": "2340"
+    },
+    {
+      "width": "1440",
+      "height": "2560"
+    },
+    {
+      "width": "720",
+      "height": "1440"
+    },
+    {
+      "width": "1080",
+      "height": "2280"
+    },
+    {
+      "width": "1080",
+      "height": "2340"
+    },
+    {
+      "width": "1080",
+      "height": "2340"
+    },
+    {
+      "width": "1440",
+      "height": "3120"
+    },
+    {
+      "width": "1080",
+      "height": "2400"
+    },
+    {
+      "width": "1080",
+      "height": "2400"
+    },
+    {
+      "width": "1440",
+      "height": "3168"
+    },
+    {
+      "width": "1080",
+      "height": "2400"
+    },
+    {
+      "width": "1080",
+      "height": "2400"
+    },
+    {
+      "width": "1080",
+      "height": "2400"
+    },
+    {
+      "width": "1080",
+      "height": "2340"
+    },
+    {
+      "width": "1080",
+      "height": "2340"
+    },
+    {
+      "width": "1080",
+      "height": "2340"
+    },
+    {
+      "width": "1080",
+      "height": "2400"
+    },
+    {
+      "width": "1080",
+      "height": "2160"
+    },
+    {
+      "width": "1080",
+      "height": "2400"
+    },
+    {
+      "width": "1080",
+      "height": "2400"
+    },
+    {
+      "width": "1536",
+      "height": "2152"
+    },
+    {
+      "width": "720",
+      "height": "1280"
+    },
+    {
+      "width": "720",
+      "height": "1280"
+    },
+    {
+      "width": "720",
+      "height": "1280"
+    },
+    {
+      "width": "720",
+      "height": "1480"
+    },
+    {
+      "width": "720",
+      "height": "1280"
+    },
+    {
+      "width": "1080",
+      "height": "1920"
+    },
+    {
+      "width": "1080",
+      "height": "1920"
+    },
+    {
+      "width": "1080",
+      "height": "1920"
+    },
+    {
+      "width": "720",
+      "height": "1480"
+    },
+    {
+      "width": "1080",
+      "height": "2400"
+    },
+    {
+      "width": "1440",
+      "height": "3088"
+    },
+    {
+      "width": "1440",
+      "height": "2960"
+    },
+    {
+      "width": "1080",
+      "height": "1920"
+    },
+    {
+      "width": "1440",
+      "height": "3040"
+    },
+    {
+      "width": "1080",
+      "height": "2400"
+    },
+    {
+      "width": "1440",
+      "height": "3040"
+    },
+    {
+      "width": "1440",
+      "height": "3200"
+    },
+    {
+      "width": "1080",
+      "height": "2400"
+    },
+    {
+      "width": "1440",
+      "height": "3200"
+    },
+    {
+      "width": "1440",
```

Comment:
- You have duplicate data here
- Let's start with 5-10 top

Reply:
that was a good one, i didn't notice, will fix asap